homeassistant/components/switcher_kis/switch.py

```diff
@@ -124,23 +122,6 @@ def is_on(self) -> bool:
         """Return True if entity is on."""
         return self._state == SWITCHER_STATE_ON
 
-    @property
-    def current_power_w(self) -> int:
-        """Return the current power usage in W."""
-        return self._device_data.power_consumption
```

Comment:
One thing I'm wondering reading this is if we should remove this. As these are not regular extra attributes, but actual properties of a switch (by model design).

<https://developers.home-assistant.io/docs/core/entity/switch#properties>

Reply:
I have no clear conclusion for this, The main reason for removing it from the switch entity, is that this sensor sends update every 4 seconds, as a separate sensor entity the user can disable it, if it is an attribute of the switch it means the switch state is updated every 4 seconds.
What do you think?

Reply:
I personally think we should make effort to wipe this property (as it's 21 uses in our codebase) from the face of the earth. But that requires architectural approval.

Reply:
In this case since this PR is a breaking change anyhow, I think it should be removed, 
I also will start a discussion about removing this property from other integrations.